twilight-model/src/guild/onboarding/mod.rs

```diff
@@ -0,0 +1,65 @@
+pub mod option;
+pub mod prompt;
+pub mod prompt_type;
```

Comment:
This module should be documented and there's no need for any sub modules, just re-export them.

```suggestion
mod option;
mod prompt;
mod prompt_type;
```